src/components/auth/RouteWithSession.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Route, RouteComponentProps } from "react-router-dom";
+import WithSession from "./WithSession";
+
+export interface RouteWithSessionProps {
+    render: (props: RouteComponentProps<any>) => React.ReactNode;
```

Comment:
All RouteComponentProps arguments have defaults, so we can simply write `RouteComponentProps`

Reply:
done!